include/JSystem/JUtility/JUTXfb/JUTXfb.h

```diff
@@ -0,0 +1,35 @@
+#ifndef __JUTILITY_JUTXFB_JUTXFB_H__
+#define __JUTILITY_JUTXFB_JUTXFB_H__
+
+#include "dolphin/types.h"
+#include "gx/GX.h"
+#include "JSystem/JKernel/JKRHeap/JKRHeap.h"
+
+class JUTXfb {
+public:
+    typedef s32 EXfbNumber;
+    JUTXfb(GXRenderModeObj* obj, JKRHeap* heap, EXfbNumber xfb_number);
+    ~JUTXfb();
+
+    void clearIndex();
+    void common_init(s32 xfb_number);
+    void delXfb(s32 index);
+    static JUTXfb* createManager(JKRHeap* heap, EXfbNumber xfb_number);
+    static void destroyManager();
+    void initiate(u16 width, u16 height, JKRHeap* heap, JUTXfb::EXfbNumber xfb_number);
+
+private:
+    void * xfb_buffer[3];
```

Comment:
might want to merge and run clang-format (won't pass new CI check otherwise)
```suggestion
    void* xfb_buffer[3];
```